slack/web/base_client.py

```diff
@@ -273,23 +279,38 @@ async def _request(self, *, http_verb, api_url, req_args):
             await session.close()
         return response
 
-    @staticmethod
-    def _get_user_agent():
-        """Construct the user-agent header with the package info,
-        Python version and OS version.
+    def _sync_send(self, api_url, req_args):
+        params = req_args["params"] if "params" in req_args else None
+        data = req_args["data"] if "data" in req_args else None
+        files = req_args["files"] if "files" in req_args else None
+        _json = req_args["json"] if "files" in req_args else None
+        headers = req_args["headers"] if "headers" in req_args else None
+        token = params.get("token") if params and "token" in params else None
+        body_params = {}
+        if params:
+            body_params.update(params)
+        if data:
+            body_params.update(data)
+
+        return self.urllib_client.api_call(
+            token=token,
+            url=api_url,
+            query_params={},
+            body_params=body_params,
+            files=files,
+            json_body=_json,
+            additional_headers=headers,
+        )
 
-        Returns:
-            The user agent string.
-            e.g. 'Python/3.6.7 slackclient/2.0.0 Darwin/17.7.0'
-        """
-        # __name__ returns all classes, we only want the client
-        client = "{0}/{1}".format("slackclient", ver.__version__)
-        python_version = "Python/{v.major}.{v.minor}.{v.micro}".format(
-            v=sys.version_info
+    def _sync_request(self, api_url, req_args):
```

Comment:
from what i understand, this method exists because even when there client is initialized with `run_async=True`, there are some reasons you would still like to make a synchronous request (like pagination iteration). if this is the intention, could we add a docstring or comment to make it more clear?

Reply:
As `_request` is an async method, we still need this method for SlackResponse iterations. I'll add docstring for mentioning this.

Reply:
I've renamed the name as `_request_for_pagination` and added docstring in https://github.com/slackapi/python-slackclient/commit/ece7a12af8c36b571b131d656c418039414542b1